test/integration/users.test.js

```diff
@@ -227,6 +227,25 @@ describe('Users', function () {
           expect(res.body).to.be.a('object')
           expect(res.body.isUsernameAvailable).to.equal(false)
 
+          return done()
+        })
+    })
+  })
+  describe('PATCH /users/identityURL', function () {
+    it('Should update the identityURL', function (done) {
+      chai
+        .request(app)
+        .patch('/users/identityURL')
+        .set('cookie', `${cookieName}=${jwt}`)
+        .send({
+          identityURL: 'identityURL'
```

Comment:
This shouldn't be allowed right? We should allow only valid links. Tomorrow someone might send `joker` here, is that a valid request body? Please see if we can use this: https://joi.dev/api/?v=17.5.0#stringurioptions